CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [3.7.1] - 2021-12-21
```

Comment:
we don't need to define the version and date here we can determine them in release time.

```suggestion
## Unreleased
```

Reply:
Commited.